clang/unittests/Analysis/FlowSensitive/TransferTest.cpp

```diff
@@ -1446,6 +1446,51 @@ TEST(TransferTest, BaseClassInitializer) {
       llvm::Succeeded());
 }
 
+TEST(TransferTest, StructModeledFieldsWithAccessor) {
+  std::string Code = R"(
+    class S {
+      int *P;
+      int *Q;
+      int X;
+      int Y;
+      int Z;
+    public:
+      int *getPtr() const { return P; }
+      int *getPtrNonConst() { return Q; }
+      int getInt(int i) const { return X; }
```

Comment:
Can you add a case that returns a reference? This produces a slightly different shape of AST (without an `LValueToRValue` cast).

Reply:
Done.